uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -202,6 +208,37 @@ def test_can_enable_false_on_unsupported_kernel_flavor(
         )
         assert (msg, "") == capsys.readouterr()
 
+    @pytest.mark.parametrize(
+        "kernel_version,meets_min_version",
+        (
+            ("3.5.0-00-generic", False),
+            ("4.2.0-00-generic", False),
+            ("4.10.0-00-generic", True),
+            ("5.0.0-00-generic", True),
```

Comment:
Let's add a case for exactly the cut-off here (i.e. 4.4), to make sure that we are drawing the line on the right side of it.

Reply:
I was using an example here, but probably best to use the expected example which should come from the live ua-contract server (4.4).

Reply:
That wasn't what this comment was about; you're responding to the one that GH has automatically hidden because you changed that line.

This request is still open.

Reply:
Thanks!